### PR TITLE
Prints out the actual error that happened

### PR DESCRIPTION
**Pull request by aaomidi**
Gives more information on https://github.com/microsoft/azuredatastudio/issues/8829

### CODE REVIEW

File: extensions/azurecore/src/account-provider/auths/azureAuth.ts [hunk 1]
```diff
@@ -231,6 +231,7 @@ export abstract class AzureAuth implements vscode.Disposable {
 					await this.refreshAccessToken(account.key, baseToken.refreshToken, tenant, resource);
 				} catch (ex) {
 					console.log(`Could not refresh access token for ${JSON.stringify(tenant)} - silently removing the tenant from the user's account.`);
+					console.log(`Actual error: ${JSON.stringify(ex?.response?.data ?? ex.message ?? ex), undefined, 2}`);
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/account-provider/auths/azureAuth.ts:234]: What's the `undefined, 2` part for? Not sure if I'm missing something obvious or what...

Comment (aaomidi) [reply extensions/azurecore/src/account-provider/auths/azureAuth.ts:234]: It prints it in a little bit more human readable format. Uses new lines and indents new parts of the JSON by 2 spaces.

Comment (aaomidi) [reply extensions/azurecore/src/account-provider/auths/azureAuth.ts:234]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify More info

Comment (Charles-Gagnon) [reply extensions/azurecore/src/account-provider/auths/azureAuth.ts:234]: @aaomidi But they're outside the stringify method? I think you closed that off too early.

Comment (aaomidi) [reply extensions/azurecore/src/account-provider/auths/azureAuth.ts:234]: OOPS!

**Pull request merged**